openstates/mi/people.py

```diff
@@ -83,8 +83,12 @@ def scrape_lower(self, chamber):
 
     def scrape_upper(self, chamber):
         url = 'http://www.senate.michigan.gov/senatorinfo_list.html'
+        photo_base_url = 'http://www.senate.michigan.gov/senatorinfo_complete.html'
+        url_to_append = 'http://www.senate.michigan.gov/'
         data = self.get(url).text
+        data2 = self.get(photo_base_url).text
```

Comment:
a more descriptive name, like photo_page might be better

Reply:
Ok @jamesturk  I will update this.